src/File/FileFinder.php

```diff
@@ -23,6 +26,34 @@ public function __construct(
 	{
 	}
 
+	private function tryToMakeRelativeExcludePath(string $excludePath, string $inPath): ?string
+	{
+		$excludePath = $this->fileHelper->normalizePath($excludePath, '/');
+		$inPath = $this->fileHelper->normalizePath($inPath, '/');
+
+		return str_starts_with($excludePath, $inPath . '/')
+			? substr($excludePath, strlen($inPath) + 1)
```

Comment:
how to fix the PHP 7.4- stan? https://github.com/phpstan/phpstan-src/actions/runs/5433198786/jobs/9880741952?pr=2503#step:12:23

Reply:
@ondrejmirtes may I have your advice here?